v2/internal/frontend/runtime/wrapper/runtime.d.ts

```diff
@@ -131,6 +131,10 @@ export function WindowSetMaxSize(width: number, height: number): void;
 // Setting a size of 0,0 will disable this constraint.
 export function WindowSetMinSize(width: number, height: number): void;
 
+// [WindowGetDimensions](https://wails.io/docs/reference/runtime/window#windowgetdimensions)
+// Gets the dimensions of the screen.
+export function WindowGetSize(): Promise<Size>;
```

Comment:
Should this be `WindowGetDimensions`?

Reply:
Yup, my mistake. Will fix this in the next public commit